LayoutTests/fast/frames/resources/self-referential-iframe.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        body, html {
+            height: 100%;
+            margin: 0;
+        }
+        .container {
+            height: 100%;
+        }
+        iframe {
+            display: block;
+            border: 0.5px solid gray;
+            width: 99.99%;
+            height: 99.99%;
+        }
+    </style>
+    <script>
+        window.addEventListener('load', () => {
+            let iframe = document.getElementsByTagName('iframe')[0];
+            let number = Math.random();
+            iframe.src = `self-referential-iframe.html?foo=${number}`;
+            window.top.postMessage(number, "*");
+        }, false);
+    </script>
+</head>
+<body>
+    <div class="container">
+    <div class="container">
+    <div class="container">
```

Comment:
Why the three nested containers?

Reply:
This is just to trigger the creation of layers to stress test the stack.